packages/website/src/components/showcase-projects.json

```diff
@@ -5,9 +5,9 @@
     "href": "https://getbootstrap.com/docs/5.2/getting-started/introduction/"
   },
   {
-    "name": "MUI",
-    "image": "img/logos/mui.svg",
-    "href": "https://mui.com"
+    "name": "Material UI",
+    "image": "img/logos/material-ui.svg",
+    "href": "https://material-ui.com/"
```

Comment:
We don't yet have https://deploy-preview-37583--material-ui.netlify.app/material-ui/ ready https://github.com/mui/material-ui/pull/37583. But we will redirect this to it, so I figure this is probably the best we can link to.